ask.sh

```diff
@@ -54,7 +54,7 @@ while ! iroot confirm; do
 
     while [ -z "$DISK" ]; do
         wallstatus install
-        DISK=$(fdisk -l | grep -i '^Disk /.*:' | sed -e "\$aother (experimental)" | imenu -l "select disk> ")
+        DISK=$(fdisk -l | grep -i '^Disk /.*:' | sed -e "\$another (experimental)" | imenu -l "select disk> ")
```

Comment:
the a is part of the sed command. this will just result in "nother"

Reply:
Yes, for some reason neovim removed that and I did't see it, sorry